hydroflow_plus/src/location/mod.rs

```diff
@@ -66,11 +67,11 @@ impl<
 }
 
 pub trait ProcessSpec<'a, D: LocalDeploy<'a> + ?Sized> {
-    fn build(&self, id: usize, builder: &'a FlowBuilder<'a, D>, meta: &mut D::Meta) -> D::Process;
+    fn build(&self, id: usize, builder: &FlowBuilder<'a, D>, meta: &mut D::Meta) -> D::Process;
 }
 
 pub trait ClusterSpec<'a, D: LocalDeploy<'a> + ?Sized> {
-    fn build(&self, id: usize, builder: &'a FlowBuilder<'a, D>, meta: &mut D::Meta) -> D::Cluster;
+    fn build(&self, id: usize, builder: &FlowBuilder<'a, D>, meta: &mut D::Meta) -> D::Cluster;
 }
 
 pub trait Location<'a>: Clone {
```

Comment:
Not sure if the `<'a>` is needed? Can probably just return `Stream<'_, ...>` below, etc?

Reply:
Or add a comment justifying why the trait has the lifetime paramterization